lib/sass/calcite-web/patterns/_accordion.scss

```diff
@@ -19,25 +19,48 @@
   }
 
   @mixin accordion-title() {
-    @include side-nav-title();
+    @include font-size(0);
+    border: 1px solid $light-gray;
+    margin: -1px 0 0 0;
+    color: $dark-gray;
```

Comment:
Can we start with `$darker-gray` here? Seems a bit light. Is there a reason we don't change that while the content is expanded? Feels a little off IMO to leave the title lighter while an area is expanded, vs, say, `$off-black`. Maybe explore making it the same weight of other active items like side-nav, dropdown item, etc. CC @bryan